### PR TITLE
Strip periods from resubmissions in addition to submissions [WA-27]

### PR DESCRIPTION
**Pull request by aednichols**
The vanilla submit mechanism sanitizes periods correctly, but the resubmit mechanism does not.

Screen Shot 2019-12-12 at 5 54 43 PM
Screen Shot 2019-12-12 at 5 55 03 PM

Follow-on to https://github.com/DataBiosphere/terra-ui/pull/1704

User reports:

https://support.terra.bio/hc/en-us/community/posts/360043380192-Error-invalid-input-while-launching-workflow
https://support.terra.bio/hc/en-us/community/posts/360048104711-Bug-Report-Relaunching-Failures-

Tickets:
https://broadworkbench.atlassian.net/browse/WA-27
https://broadworkbench.atlassian.net/browse/SUP-106

Ideally we'd put this in a shared function but I don't know how y'all do that around here and I want to be out of here by 6 😄

### CODE REVIEW

File: src/pages/workspaces/workspace/workflows/FailureRerunner.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const rerunFailures = async ({ namespace, name, submissionId, configNames
       _.map('workflowEntity')
     )(workflows)
 
-    const newSetName = `${configName}-resubmission-${new Date().toISOString().slice(0, -5).replace(/:/g, '-')}`
+    const newSetName = `${configName}-resubmission-${new Date().toISOString().slice(0, -5).replace(/[^\w]/g, '-')}` // colons in date, periods in wf name
```
Comment (zarsky-broad) [inline src/pages/workspaces/workspace/workflows/FailureRerunner.js]:
```suggestion
    const newSetName = `${configName}-resubmission-${new Date().toISOString().slice(0, -5)}`.replace(/[^\w]/g, '-') // colons in date, periods in wf name
```

Comment (zarsky-broad) [reply src/pages/workspaces/workspace/workflows/FailureRerunner.js]: You could make the `.replace(...)` part a function in `job-common.js`, like:
```js
const sanitizeWorkflowName = name => name.replace(/[^\w]/g, '-')
```
or even
```js
const buildWorkflowName = base => `${base}_{new Date().toISOString().slice(0, -5)}`.replace(/[^\w]/g, '-')
```

**Comment by aednichols** [comment]
Ready for re-review @zarsky-broad

**Review by zarsky-broad — APPROVED** [review]
Ah, I pointed you at the wrong place to put `entitySafeString`, I forgot that `jon-common` was in `components`. Could you move it to `utils.js`? Also, would you consider renaming it to be a verb, like `sanitizeEntityName` or something? I'm giving you a thumb, feel free to merge once you do.

**Pull request merged**